relayer/chains/cosmos/types.go

```diff
@@ -77,6 +31,8 @@ type clientInfo struct {
 	header          []byte
 }
 
+func (clientInfo) ibcMessageInfoIndicator() {}
```

Comment:
GitHub makes it difficult to search the content of a PR once you've collapsed some of the files -- if this satisfies an interface we can keep it, otherwise it should be removed.

Reply:
yes forgot to remove this, done.